composer.json

```diff
@@ -1,19 +1,39 @@
 {
     "name": "ebanx/magento2-gateway-ebanx",
     "description": "Oficial EBANX Magento2 Payment Gateway",
-    "version": "1.6.6",
+    "version": "1.7.6",
     "type": "magento2-module",
     "license": "Apache-2.0",
     "require": {
-        "ebanx/benjamin": "^1.18.3",
-        "magento/framework": "~100.0.0|~100.1.0|~101.0.0|~102.0.0",
-        "php": "~5.6.0|~7.0.0|~7.1.0|~7.2.0|~7.3.0"
+        "php": "~7.4",
+        "ext-gd": "*",
+        "ext-pdo": "*",
+        "ext-pdo_mysql": "*",
+        "ext-bcmath": "*",
+        "ext-intl": "*",
+        "ext-soap": "*",
+        "ext-xsl": "*",
+        "ext-zip": "*",
+        "ext-sockets": "*",
+        "ebanx/benjamin": "^1.18.3"
     },
     "require-dev": {
         "squizlabs/php_codesniffer": "3.*",
         "magento-ecg/coding-standard": ">=3.0",
         "phpunit/phpunit": "*",
-        "mockery/mockery": "dev-master"
+        "mockery/mockery": "dev-master",
+        "magento/framework": "~103.0.0"
```

Comment:
Why magento is required as an optional dependency?

Reply:
Because magento itself is already installed; there is no need to require it again - especially here. In other hand, it can be useful while developing.